vmcnet/train/default_config.py

```diff
@@ -346,6 +346,40 @@ def get_default_vmc_config() -> Dict:
                 "learning_rate": 5e-2,  # needs to be tuned with everything else
                 "learning_decay_rate": 1e-4,
             },
+            # >>>>>>>>>> from gg-min-sr-mom >>>>>>>>>>
+            "proxsr": {
+                # Learning rate params
+                "schedule_type": "inverse_time",  # constant or inverse_time
+                "learning_rate": 5e-2,  # needs to be tuned with everything else
+                "learning_decay_rate": 1e-4,
+                # ProxSR-specific params
+                "complement_decay": 0.95,
+                # Damping magnitude and type
+                "damping_type": "diag_shift",  # diag_shift or pinv
+                "damping": 0.001,
+                # Norm constraint magnitude and type
+                # NOTE: the natural norm constraint has been found to seriously reduce
```

Comment:
I'd like to remove the option for the natural constraint entirely, but could do that in a follow-up if you'd rather leave as is